aspnetcore/fundamentals/environments.md

```diff
@@ -69,6 +69,48 @@ The `Production` environment is the environment in which the application runs wh
 
 This is by no means meant to be a complete list. It's best to avoid scattering environment checks in many parts of your application. Instead, the recommended approach is to perform such checks within the application's `Startup` class(es) wherever possible
 
+## Setting the environment using the command line
+
+Setting the environment can be set from the command line when starting the app. Depending on the operating system, the method in which this is accomplished varies for either a global or session based setting.
```

Comment:
> Setting the environment can be set

The environment can be set 

Or dispense with unnecessary intro and just say something like

> The method for setting the environment depends on the operating system.